index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Product Landing Page</title>
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <!-- Header Section -->
+    <header id="header">
+      <section id="img-container">
+        <a href="#header-img">
+          <img
+            src="https://lallenfrancisl.github.io/free-code-camp-projects/productlandingpage/assets/images/original_trombones.png"
+            alt="trombonelogo"
+            id="header-img"
+          />
+        </a>
+      </section>
```

Comment:
This section lacks a heading.
You may consider adding an h2-h6 heading for best practices